tools/cli/main.js

```diff
@@ -746,6 +746,44 @@ Fiber(function () {
     rawArgs.push(term);
   }
 
+  // Prevent running meteor as root
+
+  var unixPlatforms = {
+    'darwin': true,
+    'linux': true,
+    'freebsd': true
+  };
+  var platform = process.platform;
+
+  if (_.has(unixPlatforms, platform)){
```

Comment:
This entire `unixPlatforms` check (from here up) can go away and just be replaced with:

```js
if (process.getuid) {
```

It will only be defined on `POSIX` systems.